src/file/file.service.spec.ts

```diff
@@ -154,6 +154,50 @@ describe('File Logic', () => {
     });
   });
 
+  describe('#createUpdate', () => {
+    it('should return a new upload', async () => {
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
```

Comment:
Why three identical keys? and one of them is used as parentID? 
just use the global `KEY`